### PR TITLE
Update Footer to match .com site

### PR DESCRIPTION
**Pull request by fabianrbz**
### Description

Update the footer so that it matches the .com site

### Testing instructions

Preview link: 

Screenshot 2023-11-20 at 17 48 25
Screenshot 2023-11-20 at 17 49 18
Screenshot 2023-11-20 at 17 49 00
Screenshot 2023-11-20 at 17 48 49

### Checklist 

- [ ] Review label added 
- [ ] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

File: app/_assets/stylesheets/footer.less [hunk 1]
```diff
@@ -11,38 +11,25 @@
...
-    display: inline-block;
     border-radius: 5px;
     font-size: 16px;
   }
 
   #footer-form-button {
-    width: 22%;
+    width: 25%;
     font-size: 16px;
     font-weight: normal;
```
Comment (lena-larionova) [inline app/_assets/stylesheets/footer.less:30]:
```suggestion
    font-weight: 500;
```
To match the .com site

**Comment by lena-larionova** [comment]
Between 1095px and 1147px, I get this big gap between form and list:
Screenshot 2023-11-20 at 9 18 56 AM

File: app/_assets/stylesheets/footer.less [hunk 2]
```diff
@@ -336,70 +297,19 @@
...
   .logo {
-    width: 100%;
     max-width: 120px;
-    height: 100%;
 
     a {
       display: block;
     }
     img {
       width: 100%;
```
Comment (lena-larionova) [inline app/_assets/stylesheets/footer.less:309]:
```suggestion
      width: 90%;
```
Maybe? Setting it to 90% makes it match the .com, but I don't know if it's bad practice to do this with an image.

**Comment by fabianrbz** [comment]
mobile

**Review by lena-larionova — APPROVED** [review]
LGTM!

**Pull request merged**